compiler/circle-eval-diff/driver/Driver.cpp

```diff
@@ -71,6 +71,11 @@ int entry(const int argc, char **argv)
     .default_value(false)
     .help("Print Mean Absolute PercentageError");
 
+  arser.add_argument("--print_mpeir")
```

Comment:
(optional) How about use capital letter on abbreviations? like `--print_MPEIR` It looks like lower case doesn't refer to what's abbreviated to.

Reply:
Existing options (mae and mape) are not capital letters.

It would be better to discuss this issue in another PR (or issue).